dune

```diff
@@ -1,8 +1,8 @@
+(rule
+ (target README.md.corrected)
+ (deps (package containers))
+ (action (run ocaml-mdx test %{dep:README.md} -o %{target})))
```

Comment:
btw @NathanReb I get messages like that:

```
ocaml-mdx: unknown option `-o'.
Usage: ocaml-mdx test [OPTION]... FILE
Try `ocaml-mdx test --help' or `ocaml-mdx --help' for more information.
``` and I'm not sure why?

Reply:
You need to upgrade to `mdx.1.5.0`! The `-o` option was introduced in that version.